dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -670,6 +670,14 @@ error.zosProcStart.jobname.invalid=CWWKE0943E: The z/OS STC procedure {0} failed
 error.zosProcStart.jobname.invalid.explanation=The STC job name specified by the WLP_ZOS_JOBNAME property in the server.env file is not valid. The job name must be no longer than 8 characters and must not contain a comma.
 error.zosProcStart.jobname.invalid.useraction=Verify that the job name is no longer than eight characters and that it does not contain a comma.
 
+error.server.pause.command.port.disabled=CWWKE0944E: A pause request was received for server {0}, but the server command port is disabled. The request cannot be processed. Enable the command port before requesting a pause for this server.
+error.server.pause.command.port.disabled.explanation=The server command port is required for communication with a running server.  The command port is disabled for this server, therefore the pause request could not be processed.  Note that the command port is disabled by default when a server is started as a started task on z/OS.
+error.server.pause.command.port.disabled.useraction=Enable the command port for this server in order to issue pause requests from the command line. If the server is running as a started task on z/OS, this pause can alternatively be requested from the operator console. For more information, see the knowledge center documentation.
```

Comment:
I believe the user action should just say: "Enable the command port for this server." or if we need to be more verbose: "Enable the command port for this server. See knowledge center documentation." Adding more than the the first sentence clouds what the action should really be.

Reply:
Since the other "command port disabled" messages for actions like start, stop, and dump have the action about using the operator console on z/OS, I think we should keep this in there for consistency.

Reply:
When i read the user action ... this is what i get. We are basically saying "... you are issuing the command from the command line and we detected that you had it disabled, so enable it and run it again. But just in case also know that you could have issued this command from the console .... " This implies that there should not be an issue when issuing the command is issued from the console. When the opposite is true (as far as i know) .. that is out of the box the command port is disabled by default for started tasks and enabled by default when using the command line. So, unless i misunderstood what is written, it needs to change. I would even go as far as saying that the user action for the other commands should also be updated.

Reply:
You should be able to issue commands at the console even when the command port is disabled. The command port afaik is only needed for issuing commands from the command line. So in the case of starting as a started task, if a user tries to issue the command from the command line, the command port is disabled by default, so the command can only be issued from the console.

Reply:
Thanks for clarifying. The action makes sense then.